stress/client/main.go

```diff
@@ -241,6 +242,7 @@ func performRPCs(gauge *gauge, conn *grpc.ClientConn, selector *weightedRandomTe
 			interop.DoCustomMetadata(client, grpc.WaitForReady(true))
 		}
 		numCalls++
+		atomic.AddInt32(&totalNumCalls, 1)
```

Comment:
This works, but maybe, optionally, avoid so many atomic increments via:

```go
var numCalls int64
defer func() { atomic.AddInt64(&totalNumCalls, numCalls) }()
```

Reply:
Yes, that's better, done.